docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+      - Added 802.11k RRM (Radio Resource Management) feature.
+- Multi Device Group
+  - Added Multi Device Group APIs based on IoTivity.
+    - Added group management support for devices.
+    - Added D2D communication support in a group.
+- Bluetooth
+  - Added BT 5.0 APIs for getting 2M Phy support.
+    - Added feature for BLE 5.0.
+      - http://tizen.org/feature/network.bluetooth.le.5_0
+  - Added support for fd GATT based data transfer.
+- Multi-path TCP (MPTCP)
+  - Added Multi-path TCP, which implemented in Kernel and API.
+    - Added support for Multi-path TCP functions.
+
+
+## Security
+
+### New and changed features
+
+- Security-manager
+  - DB recovery logic has been added.
+- Privilege
+  - The following privileges have been added.
+    - permission.check, updatecontrol.admin
+  - The default web privilege policy has been changed.
+    - From 5.0, web applications should declare media/externalstorage privilege if needed.
+- Default-ac-domains
+  - The System::Tools label has been added.
+    - This label will be used for system tools.
+- Askuser
+  - New APIs have been added.
+    - Getting multiple privileges as parameter.
+    - Checking permission of other application.
+- Security-analyzer
+  - The security module guide documents have been added.
+    - Security-manager, Cynara, Privilege
+
+
+## Service framework
+
+### Fixes
+
+- Account-Manager 
+  - Fixed D-Bus policies.
+  - Fixed memory leak and potential defects.
+  - Refactored code to increase code maintainability. 
+- Sync-manager 
+  - Fixed D-Bus policies.
+- Geofence-server 
+  - Fixed D-Bus policies.
+  - Fixed memory leak and potential defects.
+- LBS Server 
+  - Removed wifi dependency and added wifi-manager dependency.
+- Maps-plugin-here 
+  - Updated.
+  - Fixed memory leak and potential defects.
+  - Updated user consent.
+- Sensor 
+  - Fixed memory leak and potential defects.
+- Email 
+  - Fixed memory leak and potential defects.
+- Contact
+  - Disable unsupported charset.
+  - Fixed potential defect.
+
+
+## Web framework
+
+### New and changed features
+
+- Add feature of [App control with URI scheme]
+  - Support feature which is similar with android chrome intent.
+  - Tizen native app implicit/explicit launching by using [app-control:// ]
+  - Supporting Fallback url.
+- Implement alternative TBM back-end for web page rendering.
+  - By setting chromium-efl’s layout result can be rendered to memory surface.
+  - Using this path chromium=efl webview can be merged with given UI framework (NUI,..).
+
+
+## Lightweight Web Solution
+
+### New and changed features
+
+- WebView support for Bixby Capsule Viewer on TV has been added.
+- WebView support for DALi (NUI) has been added.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- .NET Core (Runtime)
+  - Coreclr version 2.1.1 is upgraded to 2.1.4.
+  - Corefx version 2.1.1 is upgraded to 2.1.4.
+- Xamarin.Forms
+  - Xamarin.Forms 3.2.0 which is latest stable version is supported.

```

Comment:
Can we rephrase as to maintain consistency:
Xamarin.Forms 3.2.0, which is the latest stable version, is now supported.